scripts/preprocess_spec.py

```diff
@@ -16,24 +16,63 @@ def _to_camel_case(s):
     return ''.join(_title(y) for y in s.split("_"))
 
 
-def iterate_through_operations(spec, func):
+def iterate_through_operations(spec, *funcs):
     for k, v in spec['paths'].iteritems():
         for op in _ops:
             if op in v:
-                func(v[op])
+                for func in funcs:
+                    if func(v[op], v):
+                        del v[op]
+
+
+def _has_property(prop_list, property):
+    for prop in prop_list:
+        if prop["name"] == property:
+            return True
+    return False
```

Comment:
(No action required)

Consider being consistent in returning True/False or True/None (as with ``remove_watch_operations``).